enable/tests/trait_defs/test_rgba_color_trait.py

```diff
@@ -19,6 +19,28 @@
 
 from enable.trait_defs.rgba_color_trait import RGBAColor
 
+rgba_float_dtype = np.dtype([
+    ('red', float),
```

Comment:
Nitpick-level: use "float64" instead of float, for consistency and to avoid the implicit conversion from NumPy?